### PR TITLE
docs(flicking): Document reinforcement

### PR DESCRIPTION
**Pull request by netil**
## Issue

#295, #217

## Details

Update on description
Grammatical correction

## Preferred reviewers

@naver/egjs-dev

### CODE REVIEW

File: src/flicking.js [hunk 1]
```diff
@@ -1196,7 +1192,7 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 
 		/**
 		 * Return either panel is animating or not
-		 * @ko 현재 애니메이션중인지 여부를 리턴한다.
+		 * @ko 현재 애니메이션 중인지 여부를 리턴한다.
```
Comment (jongmoon) [inline src/flicking.js]: **Suggest**

리턴한다 --> 반환한다.

File: src/flicking.js [hunk 2]
```diff
@@ -1455,7 +1451,7 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 
 		/**
 		 * Enable input events
-		 * @ko 입력 이벤트 활성
+		 * @ko 입력 이벤트를 활성한다.
```
Comment (jongmoon) [inline src/flicking.js]: **Suggest**
이벤트를 활성한다 --> 사용자 입력을 활성화한다.

File: src/flicking.js [hunk 3]
```diff
@@ -1465,7 +1461,7 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 
 		/**
 		 * Disable input events
-		 * @ko 입력 이벤트 비활성
+		 * @ko 입력 이벤트를 비활성 한다.
```
Comment (jongmoon) [inline src/flicking.js]: **Suggest**

사용자 입력을 비활성화한다.

File: src/flicking.js (hunk 2 above)
Comment (netil) [inline src/flicking.js]: To be more concise, I don't think it's necessary indicate 'user'.

Comment (jongmoon) [inline src/flicking.js]: @netil I got your point, but I think "user point" or "사용자 입력" is very common word to make meaning more clearly. only "Input" is somewhat ambiguous to me.

File: src/flicking.js [hunk 4]
```diff
@@ -8,31 +8,30 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 
 	// jscs:enable validateLineBreaks, maximumLineLength
 	/**
-	 * To build flickable UI
+	 * Component to build flickable carousel UI
 	 * @group egjs
-	 * @ko 플리킹 UI를 구성한다.
+	 * @ko 캐로셀 플리킹 UI 구성 컴포넌트
```
Comment (sculove) [inline src/flicking.js]: Could you reinforce description? like this.
https://github.com/naver/egjs/blob/master/src/infiniteGrid.js#L210-L212

**Comment by sculove** [comment]
LGTM

**Comment by jongmoon** [comment]
LGTM

**Pull request closed without merging**